### PR TITLE
Fix stale internal._dom pointers after mount

### PR DESCRIPTION
**Pull request by marvinhagemeister**

### CODE REVIEW

File: src/diff/patch.js [hunk 1]
```diff
@@ -162,4 +162,6 @@ function patchDOMElement(
 	) {
 		setProperty(dom, 'checked', tmp, oldProps.checked, false);
 	}
+
+	return dom;
```
Comment (developit) [inline src/diff/patch.js:166]: QQ: couldn't we have this set `internal._dom = dom` instead of returning? It's only used in that one spot.

Comment (marvinhagemeister) [reply src/diff/patch.js]: Good point, updated the PR with the suggestion 👍

Comment (andrewiggins) [reply src/diff/patch.js]: Yup we could. I think at the point when this pattern was established the return + set was smaller. I'd say let's leave a golf TODO and revisit this once the branch stabilizes

Comment (andrewiggins) [reply src/diff/patch.js]: Oh nevermind, Marvin fixed it :) All good.

**Pull request merged**